### DISCUSSION TITLE
Migrate to cloud from self hosted

### DISCUSSION THREAD

**Discussion by karljansson91** [directus/directus#15232] [Archive]
Hi!
I thinking about migration from my self hosted directus solution to your cloud solution. Is it possible to import/migrate my local hosted directus db to the cloud hosted? I have found no commandline tool or documentation on how to do this?.

**Comment by Ppizarrojr** [comment]
We are currently building a solution to do this yourself, but for now our team at Directus can migrate your existing db to Directus cloud for you, no problem! https://directus.io/contact/

**Comment by MrSunshyne** [comment]
@Ppizarrojr is this still in the works?

**Reply by Ppizarrojr** [reply]
Still on the roadmap, but currently, our team handles this process for you, white glove.

**Comment by cdemi** [comment]
@Ppizarrojr was this feature implemented?

**Comment by Ppizarrojr** [comment]
Hey Christopher, we still handle these migrations manually for now. Probably a roadmap item for next year.

**Reply by fayomihorace** [reply]
Hello @Ppizarrojr  is this feature already implemented ?

**Comment by Ppizarrojr** [comment]
Sign up for a directus.cloud ( http://directus.cloud/ ) account, then email support@directus.io and they can assist to import and migrate

*Pedro Pizarro*
Sales @ Directus

**Comment by ussaama** [comment]
Yes

**Comment by Zura1555** [comment]
Hi @Ppizarrojr , 

Is there any fee for the migration support?

Thank you so much

**Reply by ussaama** [reply]
nope, i migrated and they didnt charge me anything

**Reply by Zura1555** [reply]
Thank you @ussaama !
